src/views/registration-js/attendees/content.php

```diff
@@ -10,6 +10,7 @@
  * @version 4.11.0
```

Comment:
Needs a since even for whitespace changes, so we should remove them or add the since